api/hub/src/notifications.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  PrivateKey,
+  PublicKey,
+  createUserAuth,
+  Users,
+} from '@textile/hub';
+import { logger } from './helpers';
+import { parentPort, workerData } from 'worker_threads';
+
+let mailSender;
+const mailSenderIdentity = () => PrivateKey.fromString(process.env.AWF_MAILSENDER_KEY);
```

Comment:
did you test the env values being shared here?

Reply:
Yes, they work.